### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for button in "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1403]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog ,tab to and select  "Azure container registry" button.
6. Tab to button beside Docker File edit  and observe the name property using inspect.

**Actual Result:**
Name property is empty for the button 

**Narrator behaviour:**
Tab
button

**NVDA behaviour:**
button

**Expected Result:**
Name property should be simple and meaningful.
Name property for the button should be "Browse".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for containers-button-no name](https://user-images.githubusercontent.com/22956130/35684929-5453fb1e-078e-11e8-8479-76136cd38326.jpg)
![run on web app for containers-button-no name-narrator](https://user-images.githubusercontent.com/22956130/35684933-582dcc06-078e-11e8-8eff-7eb5f419d262.jpg)
![run on web app for containers-button-no name-nvda](https://user-images.githubusercontent.com/22956130/35684939-5aa1986e-078e-11e8-8e61-105371d37859.jpg)

**Comment by Eskibear** [comment]
already fixed in #1590

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1403-fixed](https://user-images.githubusercontent.com/22956130/39761117-d6d852d6-52f4-11e8-9b1f-be88296107bb.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749834
